workflows/theiacov/wf_theiacov_ont.wdl

```diff
@@ -427,5 +432,8 @@ workflow theiacov_ont {
     # QC_Check Results
     String? qc_check = qc_check_task.qc_check
     File? qc_standard = qc_check_task.qc_standard
+    # Non-flu specific outputs
+    # Non-flu specific outputs
+    Float percentage_mapped_reads = select_first([flu_stats_n_coverage.percentage_mapped_reads, stats_n_coverage.percentage_mapped_reads, 0.0])  
```

Comment:
I don't think we need the 0.0 in this select_first, but it doesn't hurt to have it.

Reply:
You also have the `# Non-flu specific outputs` line duplicated